bin/alluxio

```diff
@@ -11,7 +11,11 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-jvmdebug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo
+  echo "Flag description:"
+  echo -e "  -jvmdebug     \t If this flag specified, the env variable ALLUXIO_USER_DEBUG_JAVA_OPTS will be added to jvm flags."
```

Comment:
I see, the `format` command doesn't follow the same practices as the other commands, which breaks using generic command options with it. It's really a bug with `format` that it doesn't work with any type of generic command options. If we fix that bug, the `bin/format -debug` will start working again. To make `format` consistent with other commands, we should handle the `-s` flag within the `Java` implementation instead of at the shell level.